README.md

```diff
@@ -10,7 +10,7 @@
 
 Implementation of an Ethereum JSON RPC APIs for Hedera Hashgraph. Utilises both Hedera Consensus Nodes and Mirror nodes
 to support RPC queries as defined in
-the [JSON RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=true&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false)
+the [JSON RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/114-openrpc-doc/docs/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false)
```

Comment:
Also what would the final URL look like since this branch will get removed?

Reply:
Hmm good question.
We can switch to point to main branch before the merge or we can use the static served document from mainnet

Reply:
I like both suggestions.
Switch to use main branch for now and then once mainnet has the static doc being served we can switch that so.
If it's an easy switch i may also add it to the release flow so that it reflects the release branch it's in

Reply:
Done. Changed to main branch.